README.md

```diff
@@ -1 +1,9 @@
+[![Build Status](https://travis-ci.org/sul-dlss/trusted_checksum_repository.svg?branch=master)](https://travis-ci.org/sul-dlss/trusted_checksum_repository)
+[![Coverage Status](https://coveralls.io/repos/github/sul-dlss/trusted_checksum_repository/badge.svg)](https://coveralls.io/github/sul-dlss/trusted_checksum_repository)
+[![Dependency Status](https://gemnasium.com/badges/github.com/sul-dlss/trusted_checksum_repository.svg)](https://gemnasium.com/github.com/sul-dlss/trusted_checksum_repository)
+[![GitHub version](https://badge.fury.io/gh/sul-dlss%2Ftrusted_checksum_repository.svg)](https://badge.fury.io/gh/sul-dlss%2Ftrusted_checksum_repository)
+
+
 # README
+
+A Rails API for storing checksums. Used by the [Preservation Catalog](https://github.com/sul-dlss/preservation_core_catalog) as an external and restricted data store.
```

Comment:
maybe change line 9 to something like:

Trusted Checksum Repository is a Rails app for independently storing preservation object checksums.  Preservation Audit checks will use TCR to perform checksum validation on preserved objects.